client/signup/steps/domains/index.jsx

```diff
@@ -427,33 +429,63 @@ class DomainsStep extends Component {
 	};
 
 	getSideContent = () => {
-		return (
-			<div className="domains__domain-side-content-container">
-				{ ! this.shouldHideDomainExplainer() && this.props.isPlanSelectionAvailableLaterInFlow && (
-					<div className="domains__domain-side-content">
-						<ReskinSideExplainer
-							onClick={ this.handleDomainExplainerClick }
-							type={ 'free-domain-explainer' }
-						/>
-					</div>
-				) }
-				{ ! this.shouldHideUseYourDomain() && (
-					<div className="domains__domain-side-content">
-						<ReskinSideExplainer
-							onClick={ this.handleUseYourDomainClick }
-							type={ 'use-your-domain' }
-						/>
-					</div>
-				) }
-				{ this.shouldDisplayDomainOnlyExplainer() && (
-					<div className="domains__domain-side-content">
-						<ReskinSideExplainer
-							onClick={ this.handleDomainExplainerClick }
-							type={ 'free-domain-only-explainer' }
-						/>
-					</div>
-				) }
+		const useYourDomain = ! this.shouldHideUseYourDomain() ? (
+			<div className="domains__domain-side-content">
+				<ReskinSideExplainer onClick={ this.handleUseYourDomainClick } type={ 'use-your-domain' } />
 			</div>
+		) : null;
+
+		return (
+			<ProvideExperimentData
+				name="calypso_signup_domain_mobile_brower_chrome_added"
+				options={ {
+					isEligible: isMobile() && 'onboarding' === this.props.flowName,
+				} }
+			>
+				{ ( isLoading, experimentAssignment ) => {
+					if ( isLoading ) {
+						return null;
+					}
+
+					if ( experimentAssignment?.variationName === 'treatment' ) {
+						return (
+							<div className="domains__domain-side-content-container domains__domain-side-content-container-mobile-experiment">
+								<div className="domains__domain-side-content-container-browser-chrome">
+									<span></span>
+									<span></span>
+									<span className="domains__domain-side-content-container-browser-chrome-url">
+										https://{ this.props.translate( 'yoursitename.com' ) }
```

Comment:
Should we remove the .com from the translate function to make it a bit easier for the translators?

Reply:
Good idea!